File: MODUL 5/Soal 4/PRAK404-2210817310008-Muh-Haekal-Barera.py
def pilihan ():
    print("\nPilih Program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n")
    nilai = int(input('Masukkan Pilihan : '))
    if(nilai >= 1 and nilai <= 4) :
        a = float(input('\nMasukkan Nilai Pertama : '))
        b = float(input('\nMasukkan Nilai Kedua : '))
        if (nilai == 1) :
            hasil = a + b
            print("\nHasil Penjumlahan Antara %.2f dan %.2f Adalah %.2f\n" %(a,b,hasil))
            pilihan()
        elif (nilai == 2) :
            hasil = a - b
            print("\nHasil Pengurangan Antara %.2f dan %.2f Adalah %.2f\n" %(a,b,hasil))
            pilihan()
        elif(nilai == 3) :
            hasil = a * b
            print("\nHasil Perkalian Antara %.2f dan %.2f Adalah %.2f" %(a,b,hasil))
            pilihan()
        elif(nilai == 4) :
            hasil = a / b
            print("\nHasil Pembagian Antara %.2f dan %.2f Adalah %.2f" %(a,b,hasil))
            pilihan()
    elif (nilai == 5) :
        print("\nTerima kasih telah menggunakan kalkulator Muh. Haekal Barera\n")
    else :
        print("Input Anda Salah, Silahkan Coba Lagi \n")
        pilihan()
pilihan()

File: MODUL 5/Soal 4/PRAK404-2210817310008-Muh-Haekal-Barera.c
#include <stdio.h>

int main()
{
    int nilai;
    float a, b, hasil;
pilihan:
    printf("\nPilih Program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
    printf("Masukkan Pilihan : ");
    scanf("%d", &nilai);
    if (nilai >= 1 && nilai < 5)
    {
        printf("Masukkan Nilai Pertama : ");
        scanf("%f", &a);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f", &b);
        if (nilai == 1)
        {
            hasil = a + b;
            printf("\nHasil Penjumlahan Antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            goto pilihan;
        }
        else if (nilai == 2)
        {
            hasil = a - b;
            printf("\nHasil Pengurangan Antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            goto pilihan;
        }
        else if (nilai == 3)
        {
            hasil = a * b;
            printf("\nHasil Perkalian Antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            goto pilihan;
        }
        else if (nilai == 4)
        {
            hasil = a / b;
            printf("\nHasil Pembagian Antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            goto pilihan;
        }
    }
    else if (nilai == 5)
    {
        printf("\nTerima kasih telah menggunakan kalkulator Muh. Haekal Barera\n");
    }
    else
    {
        printf("Input Anda Salah, Silahkan Coba Lagi \n");
    }
    return 0;
}
